azkaban-common/src/test/java/azkaban/executor/container/watch/KubernetesWatchTest.java

```diff
@@ -297,6 +302,12 @@ public void testFlowManagerListenerInitContainerFailure() throws Exception {
     // Setup a FlowUpdatingListener
     Props azkProps = new Props();
     FlowStatusManagerListener updatingListener = flowStatusUpdatingListener(azkProps);
+    // Verify EXECUTION_STOPPED flow life cycle event is emitted
+    updatingListener.addListener((event) -> {
```

Comment:
The code is duplicated at 3 places. Put it in a common method and call it here.

Reply:
addressed the comment